src/fsharp/FSharp.Build/Fsc.fs

```diff
@@ -195,6 +196,11 @@ type public Fsc () as this =
 
         builder.AppendSwitchIfNotNull("--warnaserror:", warningsAsErrorsArray, ",")
 
+        // WarningsNotAsErrors
+        match warningsNotAsErrors with
+        | null -> ()
+        | _ -> builder.AppendSwitchIfNotNull("--warnaserror-:", warningsNotAsErrors.Split([|' '; ';'; ','|], StringSplitOptions.RemoveEmptyEntries), ",")
```

Comment:
Won't we get a NullReferenceException if \<WarningsNotAsErrors\> isn't set?

Reply:
We are not checking null for any other property and default value is null for each. This pattern is used few lines above (see https://github.com/Microsoft/visualfsharp/pull/4562/files#diff-bcb4028a918d13bbab2d0c1c15c97e6aR178, and default value of `disabledWarnings` is set to `null` at the top)

Reply:
Sorry I'm blind - I didn't see the fact that we were in a match where we explicitly check for null